web-common/src/features/dashboards/filters/Filters.svelte

```diff
@@ -128,8 +128,10 @@ The main feature-set component for dashboard filters
 
   $: if (!$topListQuery?.isFetching && searchText != "") {
     const topListData = $topListQuery?.data?.data ?? [];
-    searchedValues =
-      topListData.map((datum) => datum[activeDimensionName]) ?? [];
+    const activeColumn =
```

Comment:
We should move the calculation of the active column outside of this since this runs every keystroke.

Reply:
Done